### DISCUSSION TITLE
Problem in Icon Loading

### DISCUSSION THREAD

**Discussion by TakshPanchal** [nandorojo/solito#264] [Q&A]
I am using [drawer navigation](https://reactnavigation.org/docs/drawer-navigator) from react nativation. I am facing issues in web. Getting this error.
```
../../node_modules/@react-navigation/drawer/lib/module/views/assets/toggle-drawer-icon.png
TypeError: unsupported file type: undefined (file: undefined)
```

I found same issue while doing this in [with-tailwind](https://github.com/nandorojo/solito/tree/master/example-monorepos/with-tailwind) template.

**Comment by TakshPanchal** [comment]
This might be the same issue #262

**Comment by TakshPanchal** [comment]
hey @nandorojo can you help me here :) please.

**Comment by AlvSovereign** [comment]
In web, ensure you're importing the screen/component needed that doesn't contain any references to `@react-navigation`. Your screens should only have imports responsible for the UI. Also, make sure your import statement is direct to the component/screen you want to render. 

I had this issue when I was trying to import a screen from an intext file that also exported other components that contained an import from `@react-navigation`
